lib/src/cli.rs

```diff
@@ -116,6 +116,11 @@ pub(crate) struct StatusOpts {
     /// Only display status for the booted deployment.
     #[clap(long)]
     pub(crate) booted: bool,
+    
+    /// Condense status into most important info, to reduce eye fatigue 
```

Comment:
I think we can probably just go ahead and default to human-readable if we're writing to a tty by default.

That avoids the need to call it anything at all, and avoids "typing fatigue" from a potentially frequently used CLI option.

Arguably then, we could also simultaneously switch to JSON by default if we're not on a tty.

But, we probably do need a general option here like:

```
enum OutputFormat {
  HumanReadable,
  YAML,
  JSON
}
```
or so and 

```
pub(crate) format Option<OutputFormat>
```

or so.

Reply:
https://github.com/containers/bootc/pull/639